modules/application/pages/index.adoc

```diff
@@ -8,7 +8,41 @@ You can easily analyze all execution data of your processes to better understand
 
 It is also possible to set time-based goals (Business Goals) to verify the achievement of your business objectives.
 
+== Prerequisites
+
+* Internet connexion
+
+* Any web browser
```

Comment:
Any supported browser in latest version

Reply:
list should be available somewhere in a prerequisite page

Reply:
text modified